packages/belt-holes/src/BeltHoles_List.rei

```diff
@@ -10,3 +10,10 @@ let groupByString: (list('a), 'a => string) => Map.String.t(list('a));
     and grouping the results according to values returned. */
 let groupBy:
   (list('v), Map.id('k, 'c), 'v => 'k) => Map.t('k, list('v), 'c);
+
+/** Checks a list to contain some value */
+let contains: ('a, list('a)) => bool;
```

Comment:
Belt (and thus BeltHoles) use "`t`-first" arguments order. So it should be `let contains: (list('a), 'a) => bool;`